docs/man/man1/halui.1

```diff
@@ -546,16 +546,16 @@ pin for setting the scale on changing the Rapid Override
 .B halui.rapid\-override.value\fR float out \fR
 current Rapid Override value
 
-.SS estop
+.SS E-stop
```

Comment:
Just that ".SS" is non-syntactical and for the human. We have not yet decided about how we want to spell E-stop - we have estop, e-stop, estop, Estop, E-Stop, EStop, ESTOP, E-STOP and E-stop throughout the documentation.  I find E-stop to be the most elegant and would like to use that throughout.

Reply:
Okay it is not 100% consistent now.
But starting with an uppercase letter makes it not more consistent ;-)
![image](https://user-images.githubusercontent.com/67957916/199717305-da990a43-af35-49ea-8740-9ad5c92b431b.png)

Reply:
It is home and not home-all, rapid override and not rapid-override - but maybe it would convince you to think about the translations - that would be 'Notaus' and 'Referenzierung', not the syntax that happens to be in English.

Reply:
Capital and hyphen it is :)

Reply:
> It is home and not home-all, rapid override and not rapid-override 

That's what I was meaning with "not 100% consistent" ;-)
To have it nice and consistent I would suggest either

- ...
- rapid override -> Rapid Override 
- estop -> E-stop
- home -> Homing
-  ...

or leave it as it is to have it just consistent.

Reply:
Ah, fine, will happily do that.

Reply:
Thinking to have addressed that with my latest push.